examples/step-60/doc/results.dox

```diff
@@ -0,0 +1,415 @@
+Neumann boundary, change the parametrization, the function values and the
+refinements.
+
+However, non aligned meshes are complex to handle, as you shall see after some
+changes. For example set the following parameters:
+
+@code
+subsection Distributed Lagrange<1,2>
+  set Homogeneous Dirichlet boundary ids           = 0, 1, 3
+  subsection Embedded configuration
+    set Function constants  = R=.2, Cx=.5, Cy=.4
+    set Function expression = R*sin(2*pi*x)+0.5*R*sin(4*pi*x)+Cx; 1.5*R*cos(2*pi*x)+Cy
+  end
+end
+@endcode
+
+Keeping the default refinement options you shall notice some glitch in the upper part
+of this hot air balloon shape. Why? What's the difference with the following?
+
+<TABLE WIDTH="60%" ALIGN="center">
+  <tr>
+    <td ALIGN="center">
+      <img src="https://www.dealii.org/images/steps/developer/step-60.2_no_grid.png" alt="">
+    </td>
+    <td ALIGN="center">
+      <img src="https://www.dealii.org/images/steps/developer/step-60.2_grid.png" alt="">
+    </td>
+  </tr>
+</table>
+
+<h4> Preconditioner </h4>
+
+At the moment, we have no preconditioner on the Schur complement. This is ok for
+two dimensional problems, where a few hundred iterations bring the residual down
+to the machine precision, but it's not going to work in three dimensions.
+
+It is not obvious what a good preconditioner would be here. The physical problem
+we are solving with the Schur complement, is to associate to the Dirichlet data
+$g$, the value of the Lagrange multiplier $\lambda$. $\lambda$ can be
+interpreted as the *jump* in the normal gradient that need to be imposed on $u$
+across $\Gamma$, in order to obtain the Dirichlet data $g$.
+
+So $S$ is some sort of Neumann to Dirichlet map, and we would like to have a
+good approximation for the Dirichlet to Neumann map. A possibility would be to
+use a Boundary Element approximation of the problem on $\Gamma$, and construct a
+rough approximation of the hyper-singular operator for the Poisson problem
+associated to $\Gamma$, which is precisely a Dirichlet to Neumann map.
+
+<h4> Parallel Code </h4>
+
+The simple code proposed here can serve as a starting point for more
+complex problems which, to be solved, need to be run on parallel
+code, possibly using distributed meshes (see step-17, step-40, and the
+documentation for parallel::shared::Triangulation and
+parallel::distributed::Triangulation ).
+
+When using non-matching grids in parallel a problem arises:
+to compute the matrix $C$ a process needs information about both
+meshes on the same portion of real space but, when working with
+distributed meshes, this information may not be available.
+
+Various strategies can be implemented to tackle this problem:
+
+- distribute the two meshes so that this constraint is satisfied
+
+- use communication for the parts of real space where the constraint is not
+  satisfied
+
+- use a shared triangulation for the embedding space, and a distributed
+  triangulation for the emdedded configuration
```

Comment:
add a period.

Could one also reverse this strategy?

Reply:
Yes, but why? The embedding domain is bigger (by definition), so you'll most likely want to distribute the one that is more expensive, and keep as shared the one that is less expensive.

Reply:
By the way, for 9.1, I'm hoping that @GivAlz will have finished with his ongoing project, that would allow one to distributed grids on both Triangulations.

Reply:
(I'll add a sentence to the tutorial about the reverse strategy).